Example/BonMot/BONSpecialGenerator.swift

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env swift
+
+//
+//  BONSpecialGenerator.swift
+//  BonMot
+//
+//  Created by Zev Eisenberg on 7/16/15.
+//
+
+import Foundation
+import AppKit
+
+// Please keep this array sorted
+let specialCharacters: [unichar] = [
+    0x0009,
+    0x000A,
+    0x0020,
+    0x00A0,
+    0x2002,
+    0x2003,
+    0x2007,
+    0x2009,
+    0x200A,
+    0x200B,
+    0x2011,
+    0x2012,
+    0x2013,
+    0x2014,
+    0x2026, // HORIZONTAL ELLIPSIS
```

Comment:
#req No need to comment, unless you want to go through and comment all of them. I prefer to just look up the appropriate code in the .m file if you need it. (Commenting each one could be nice, though - what do you think?)